dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESReadOnlyMonitorTest.java

```diff
@@ -0,0 +1,191 @@
+
+            Thread.sleep(100);
+
+            checkLargeMessageSent(indiciesInfo, user);
+            assertEquals(false, ESIndexUtil.isReadOnly(indiciesInfo.getWorking(), indiciesInfo.getLive()));
+        } finally {
+            putReadOnly(indiciesInfo.getWorking(), false);
+        }
+    }
+
+    /**
+     * Method to Test: {@link ESReadOnlyMonitor#start(ReindexEntry, String)}
+     * When: If call start again after the first call is finished
+     * Should: should sent the message again
+     *
+     * @throws DotDataException
+     */
+    @Test
+    public void shouldSendLargeMessageTwice() throws DotDataException, DotSecurityException, InterruptedException {
+        final ReindexEntry reindexEntry = mock(ReindexEntry.class);
+        final String cause = "cause";
+
+        final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+
+        final Role adminRole = mock(Role.class);
+        when(roleAPIMock.loadCMSAdminRole()).thenReturn(adminRole);
+
+        final User user = mock(User.class);
+        when(user.getUserId()).thenReturn("1");
+
+        when(roleAPIMock.findUsersForRole(adminRole)).thenReturn(list(user));
+
+        try {
+            putReadOnly(indiciesInfo.getWorking(), true);
+            putReadOnly(indiciesInfo.getLive(), false);
+
+            esReadOnlyMonitor.start(reindexEntry, cause);
+            checkLargeMessageSent(indiciesInfo, user);
+
+            esReadOnlyMonitor.start(reindexEntry, cause);
+            checkLargeMessageSent(indiciesInfo, user);
+            assertEquals(false, ESIndexUtil.isReadOnly(indiciesInfo.getWorking(), indiciesInfo.getLive()));
+        } finally {
+            putReadOnly(indiciesInfo.getWorking(), false);
+        }
+    }
+
+    private void checkLargeMessageSent(IndiciesInfo indiciesInfo, User user) throws InterruptedException {
+        Thread.sleep(100);
+
+        final SystemMessageBuilder messageReadonly = new SystemMessageBuilder()
+                .setMessage("At least one of the Elasticsearch current indices are in read only mode")
+                .setSeverity(MessageSeverity.ERROR)
+                .setType(MessageType.SIMPLE_MESSAGE)
+                .setLife(TimeUnit.SECONDS.toMillis(5));
+
+        verify(systemMessageEventUtilMock).pushMessage(
+                messageReadonly.create(),
+                list(user.getUserId())
+        );
+
+        final SystemMessageBuilder messageWriteModeAgain = new SystemMessageBuilder()
+                .setMessage("Elasticsearch current indices are in write mode again")
+                .setSeverity(MessageSeverity.ERROR)
+                .setType(MessageType.SIMPLE_MESSAGE)
+                .setLife(TimeUnit.SECONDS.toMillis(5));
+
+        verify(systemMessageEventUtilMock).pushMessage(
+                messageWriteModeAgain.create(),
+                list(user.getUserId())
+        );
+    }
+
+    private static AcknowledgedResponse putReadOnly(final String indexName, final boolean value) {
```

Comment:
I would rename to `setReadOnly`

Reply:
done https://github.com/dotCMS/core/pull/18604/commits/fc979e0e1d04475951dbb213b379b17022474a8e#diff-8d99314a049a8e40fb6783cfaf992d60R173